okhttp-clients/src/test/java/com/palantir/remoting3/okhttp/QosIoExceptionInterceptorTest.java

```diff
@@ -60,7 +60,7 @@ public void test308WithoutLocation() throws Exception {
 
     @Test
     public void test308WithLocation() throws Exception {
-        URL url = new URL("http://foo");
+        URL url = new URL("http://127.0.0.1");
```

Comment:
Should we not test with a different URL than the request?

Reply:
done

Reply:
https://github.com/palantir/http-remoting/pull/534